Flutter/cartesian-charts/axis-customization.md

```diff
@@ -1383,7 +1383,7 @@ Here you need to specify the [`minimum`](https://pub.dev/documentation/syncfusio
 
 ## Multi-level labels
 
-The multi-level labels in the Cartesian chart are used to categorize axis labels and they can be added to the chart axis by using the [`multiLevelLabels`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartAxis/multiLevelLabels.html) property. The below properties are used to add the multi-level label.
+The multi-level labels in the Cartesian chart are used to categorize axis labels and they can be added to the chart axis by using the [`multiLevelLabels`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/NumericMultiLevelLabel-class.html) property. The below properties are used to add the multi-level label.
```

Comment:
Use CategoryMultiLevelLabel

Reply:
@LavanyaGowtham2021,

Here, **multiLevelLabels** is a property name and it is not a class name. So, I have updated NumericMultiLevelLabel class name. Because, in the below mentioned start, end and text properties was in the NumericMultiLevelLabel class name.

Commit link - https://github.com/syncfusion-content/flutter-docs/pull/927/commits/1e620132c25368961d6fbb3f6ae58f14fb62fe6a

Regards,
Lokesh P.